client/viame-web-common/components/AttributesPanel.vue

```diff
@@ -663,23 +638,23 @@ export default defineComponent({
   }
 }
 .attribute-name {
-  font-size:.8em;
-  max-width:50%
+  font-size: 0.8em;
+  max-width: 50%;
 }
 .border-highlight {
-   border-top: 1px solid gray;
-   border-bottom: 1px solid gray;
-   color: white;
-   font-weight: bold;
-   font-size: .90em;
-   min-height: 45px;
- }
- .scroll-section {
-   overflow-y: auto;
-   overflow-x: hidden;
- }
- .confidence {
-     min-height: 40px;
-
-   }
+  border-top: 1px solid gray;
+  border-bottom: 1px solid gray;
+  color: white;
+  font-weight: bold;
+  font-size: 0.9em;
+  padding: 4px 10px;
+  background-color: #272727;
```

Comment:
padding and background color are the only actual changes here.  everything else is autoformatted.